glide.yaml

```diff
@@ -76,6 +76,7 @@ testImport:
   subpackages:
   - parallel-exec
   - update-license
+- package: go.uber.org/fx
```

Comment:
Oh, sweet - didn't fully process that this wouldn't require a compile-time Fx dependency.